src/main.rs

```diff
@@ -48,7 +52,10 @@ fn main() -> Result<(), error::Error> {
         .set_verbose(opts.verbose)
         .get_sections(&opts.branch)?;
     println!("Checking Cargo manifest");
-    let manifest = cargo::Parser::from_manifest_path("./Cargo.toml")?;
+    let path = opts
+        .cargo_toml
+        .unwrap_or_else(|| String::from("./Cargo.toml"));
```

Comment:
Are we looking for [unwrap_or](https://doc.rust-lang.org/std/option/enum.Option.html#method.unwrap_or) here ?

Reply:
Yeah, initially I wanted to go with `unwrap_or`, but `clippy_pedantic` suggests `unwrap_or_else`

Reply:
Oh that's funny, I gotta give it a try on my computer, I wonder why clippy says that Oo

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#or_fun_call

"The function will always be called and potentially allocate an object acting as the default."

Oh I didn't know that, TIL ! thank you for pointing me to it ! :D